lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -452,6 +452,18 @@ inventory:
     config_get: "show inventory"
     config_get_token: ["TABLE_inv", "ROW_inv", 0, "vendorid"]
 
+ntp_server:
+  server:
+    config_get: "show running-config all | include 'ntp server'"
+    config_get_token: '/(?:[0-9]{1,3}\.){3}[0-9]{1,3}/'
```

Comment:
I'd prefer to see this regexp anchored a bit more - perhaps something like:

`config_get_token: '/ntp server ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/'`